ethcore/src/types/executed.rs

```diff
@@ -20,13 +20,14 @@ use util::numbers::*;
 use util::Bytes;
 use trace::{Trace, VMTrace};
 use types::log_entry::LogEntry;
+use types::state_diff::StateDiff;
 use ipc::binary::BinaryConvertError;
 use std::fmt;
 use std::mem;
 use std::collections::VecDeque;
 
 /// Transaction execution receipt.
-#[derive(Debug, PartialEq, Clone, Binary)]
+#[derive(Debug, PartialEq, Clone)]
```

Comment:
this was discussed with @NikVolf and decided that although it will eventually need to be binary, the infrastructure is not yet in place to easily allow that and so it will remain for another PR.